### PR TITLE
Task/param check

### PR DESCRIPTION
**Pull request by mclarsen**

### CODE REVIEW

**Comment by mclarsen** [comment]
plots no longer have member 'params'. 'plots/p1/params/field' becomes 'plots/p1/field'. Its simpler and better, but this will break things.

**Comment by mclarsen** [comment]
@cyrush what do you think? I converted vtkh filters as a first step.

File: src/ascent/runtimes/flow_filters/ascent_runtime_vtkh_filters.cpp [hunk 1]
```diff
@@ -1137,12 +1204,30 @@ DefaultRender::verify_params(const conduit::Node &params,
...
+
+    std::string surprises = surprise_check(valid_paths, ignore_paths, params);
+
+
+    // parse render surprises
+    if(params.has_path("renders"))
     {
-        info["errors"].append() = "Missing required string parameter 'image_prefix'";
-        res = false;
+      const conduit::Node renders_node = params["renders"];
```
Comment (cyrush) [inline src/ascent/runtimes/flow_filters/ascent_runtime_vtkh_filters.cpp]: found case w/ missed node ref (to avoid copy)

**Review by cyrush — APPROVED** [review]
looks good found a missed case where you could use a ref

**Pull request merged**